common/src/test/java/com/microsoft/identity/common/unit/AzureActiveDirectoryB2CAuthorityTest.java

```diff
@@ -0,0 +1,32 @@
+package com.microsoft.identity.common.unit;
+
+import com.microsoft.identity.common.internal.authorities.Authority;
+import com.microsoft.identity.common.internal.authorities.AzureActiveDirectoryB2CAuthority;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+
+@RunWith(RobolectricTestRunner.class)
+public class AzureActiveDirectoryB2CAuthorityTest {
+
+    private static final String POLICY_NAME = "policyName";
+    private static final String TENANT_NAME = "tenantName";
+    private static final String TENANT_ID = "tenantID";
+    private static final String AAD_B2C_HOSTNAME = "azureADB2CHostname";
+    private static final String[] B2CAuthorityUrls = new String[]{
+            "https://"+AAD_B2C_HOSTNAME+"/tfp/"+TENANT_NAME+"/"+POLICY_NAME,
+            "https://"+TENANT_NAME+".b2clogin.com/tfp/"+TENANT_ID+"/"+POLICY_NAME
+    };
+
+    @Test
+    public void testGetB2CPolicyName() {
+        for (final String authorityUrl : B2CAuthorityUrls) {
```

Comment:
Instead of having a for-loop here...you may want to write a Parameterized test instead

Reply:
43f69c5bd5c9dbb87f235509c77ac18aa1ed42a9